src/java.desktop/share/classes/java/awt/FlowLayout.java

```diff
@@ -51,31 +51,38 @@
  * <li>{@link #TRAILING TRAILING}
  * </ul>
  * <p>
- * For example, the following picture shows an applet using the flow
+ * For example, the following picture shows a window using the flow
  * layout manager (its default layout manager) to position three buttons:
  * <p>
- * <img src="doc-files/FlowLayout-1.gif"
+ * <img src="doc-files/FlowLayout-1.png"
  * ALT="Graphic of Layout for Three Buttons"
  * style="margin: 7px 10px;">
  * <p>
- * Here is the code for this applet:
+ * Here is the code for this program:
  *
- * <hr><blockquote><pre>
- * import java.awt.*;
- * import java.applet.Applet;
+ * <hr>
+ * {@snippet  lang='java':
```

Comment:
```suggestion
 * {@snippet lang='java':
```
Not really an issue, however, other instances have just one space.